exporter/dorisexporter/exporter_logs.go

```diff
@@ -12,7 +12,7 @@ import (
 
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/pdata/plog"
-	semconv "go.opentelemetry.io/otel/semconv/v1.25.0"
+	semconv "go.opentelemetry.io/otel/semconv/v1.37.0"
```

Comment:
Is there a reason not to use [v1.38.0](https://pkg.go.dev/go.opentelemetry.io/otel@v1.39.0/semconv/v1.38.0)?

```suggestion
	semconv "go.opentelemetry.io/otel/semconv/v1.38.0"
```